flash/text/question_answering/data.py

```diff
@@ -298,7 +496,111 @@ def from_dicts(
             doc_stride: The stride amount to be taken when splitting up a long document into chunks.
 
         Returns:
-            The constructed data module.
+            The constructed :class:`~flash.text.question_answering.data.QuestionAnsweringData`.
+
+        Examples
+        ________
+
+        .. testsetup::
```

Comment:
Let's have this in the main doctest instead

Reply:
Could you explain this further ?
I am not familiar with the new doctests code fully.